components/NavBar.vue

```diff
@@ -6,11 +6,11 @@
       app
       flat
       height="85"
-      class="mainNav bgViolet"
+      class="mainNav bgBurgundy"
     >
       <v-toolbar-title>
-        <NuxtLink class="link-box px-3" to="/">
-          <Logo alternative white has-custom-height custom-height="65" />
+        <NuxtLink class="link-box px-3 ml-6git" to="/">
```

Comment:
Aqui, acho que escapou um comando do git do terminal, kkkkkkk. Mas também recomendo não manter a margem só à esquerda. Conforme o Figma, seria interessante se adicionássemos uma margem à esquerda e à direita da `NavBar` que se alinhasse com o container das páginas (acredito que foi isso que você pensou em fazer).

Vou dar uma sugestão no próximo comentário de como fazer isso! Segue, abaixo, a sugestão para reverter as mudanças dessa linha.
```suggestion
        <NuxtLink class="link-box px-3" to="/">
```

Reply:
Existe um componente do Vuetify chamado `v-container`. Ele é usado nas páginas para, dinamicamente, ajustar esses "espaços laterais" que comentei anteriormente. Como queremos que a `NavBar` tenha os mesmos espaços, então a solução é adicionar o `v-container` no componente, para a versão desktop.

Quando fazemos isso, no entanto, acontece um desalinhamento dos componentes dentro da `NavBar` (o _container_ é um componente que, em teoria, "cresce para baixo", daí, os links para as páginas todos são dispostos como uma lista vertical). A solução pra isso é adicionar, dentro do `v-container`, uma `v-row` com alinhamento central (assim, especificamos que queremos os itens dispostos em uma linha, não em várias linhas).

Logo, a parte do código do componente `NavBar` em que temos
```diff
    <v-app-bar
      v-if="!isMobile"
      fixed
      app
      flat
      height="85"
      class="mainNav bgBurgundy"
    >
      <v-toolbar-title>
        <NuxtLink class="link-box px-3" to="/">
          <Logo alternative isNavBar has-custom-height custom-height="65" />
        </NuxtLink>
      </v-toolbar-title>
      <v-spacer />

      <template v-for="(link, i) in links">
        <NuxtLink
          v-if="!link.ehExterno"
          :key="`link-${i}`"
          class="link-box white--text px-5"
          :to="link.url"
        >
          {{ link.nome }}
        </NuxtLink>
        <a
          v-else
          :key="`link-${i}`"
          :href="link.url"
          class="white--text px-5"
          target="_blank"
        >
          {{ link.nome }}
        </a>
      </template>
    </v-app-bar>
```

seria adaptada assim:

```diff
    <v-app-bar
      v-if="!isMobile"
      fixed
      app
      flat
      height="85"
      class="mainNav bgBurgundy"
    >
+     <v-container>
+      <v-row align="center">
          <v-toolbar-title>
            <NuxtLink class="link-box px-3" to="/">
              <Logo alternative isNavBar has-custom-height custom-height="65" />
            </NuxtLink>
          </v-toolbar-title>
          <v-spacer />

          <template v-for="(link, i) in links">
            <NuxtLink
              v-if="!link.ehExterno"
              :key="`link-${i}`"
              class="link-box white--text px-5"
              :to="link.url"
            >
              {{ link.nome }}
            </NuxtLink>
            <a
              v-else
              :key="`link-${i}`"
              :href="link.url"
              class="white--text px-5"
              target="_blank"
            >
              {{ link.nome }}
            </a>
          </template>
+       </v-row>
+     </v-container>
    </v-app-bar>
```